### PR TITLE
Move to common source blocks for SDRs between scanner and birdseye.

### PR DESCRIPTION
**Pull request by anarkiwi**

### CODE REVIEW

**Comment by anarkiwi** [comment]
Do not merge yet - still testing with each SDR.

**Comment by anarkiwi** [comment]
Now OK to merge (if passes!).

**Review by cglewis — APPROVED** [review]

**Pull request merged**
